### PR TITLE
update and adding gas station doc

### PR DESCRIPTION
**Pull request by EmpieichO**
Fixed one miden/index image link + added zkevm-gas-station.md

### CODE REVIEW

File: docs/zkEVM/get-started/zkevm-gas-station.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "safeLow":1,
+    "standard":1,
+    "fast":1,
+    "fastest":1,
+    "blockTime":2,
+    "blockNumber":308789
+}
+```
+
+- {`safelow`, `standard`, `fast`, `fastest`} are gas prices in Gwei, you can use these prices before sending the transaction off to Polygon zkEVM, depending upon your needs
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/zkevm-gas-station.md]:
```suggestion
- {`safelow`, `standard`, `fast`, `fastest`} are gas prices in Gwei, you can use these prices before sending the transaction off to Polygon zkEVM, depending upon your needs.
```

File: docs/zkEVM/get-started/zkevm-gas-station.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- `blockTime`, in seconds, gives the average block time of the network
+- `blockNumber` provides the information of the latest block mined when the recommendation was made
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/zkevm-gas-station.md]:
```suggestion
- `blockTime`, in seconds, gives the average block time of the network.
- `blockNumber` provides the information of the latest block mined when the recommendation was made.
```

File: docs/zkEVM/get-started/zkevm-gas-station.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
+<!-- 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem'; -->
+
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/zkevm-gas-station.md]:
```suggestion
```

File: docs/miden/index.md [hunk 4]
```diff
@@ -4,6 +4,6 @@
 Polygon Miden is an in-development zero-knowledge (ZK) rollup running the Miden VM, a virtual machine that prioritizes ZK-friendliness over EVM compatibility.
 </div>
 <div class="flex-figure-right">
-<img src="../img/miden/miden.svg" class="figure figure-right" alt="" />
+<img src="../../img/miden/miden.svg" class="figure figure-right" alt="" />
```
Comment (kmurphypolygon) [inline docs/miden/index.md]: Really? this is working on the frontend? It looks like this will break it...

**Review by kmurphypolygon — CHANGES_REQUESTED** [review]
Please accept the suggestions and consider the image comment.

**Review by EmpieichO — COMMENTED** [review]
Changes accepted

**Review by kmurphypolygon — APPROVED** [review]

**Pull request merged**